### DISCUSSION TITLE
Win+X shows both PowerShell and Terminal (each one twice)

### DISCUSSION THREAD

**Discussion by larsen0815** [valinet/ExplorerPatcher#3399] [Q&A]
Hi,

I have recently started using Windows 11 23H2 and only then have installed ExplorerPatcher. A little while later I discovered that the Win+X menu contains both a link to PowerShell and a link to Terminal (each one twice because of "as admin").

Also using 7+ Taskbar Tweaker and OpenShell.

Don't know what caused this and am looking for clues. Couldn't find a setting in EP. Am I missing something there or is this probably not coming from EP?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by larsen0815** [comment]
Version 23H2 (Build 22631.3737)
EP: 22621.3527.65.1
Settings: see screenshots
Other software: see above

![capture_20240619_164642](https://github.com/valinet/ExplorerPatcher/assets/2345512/8b0bf7c4-bbea-41f2-a987-095e77865fe6)
![capture_20240619_164647](https://github.com/valinet/ExplorerPatcher/assets/2345512/5f636737-46ed-4bb7-8a4b-15f2409b2b4f)
![capture_20240619_164648](https://github.com/valinet/ExplorerPatcher/assets/2345512/55ff5a9c-1c4f-4ff4-9068-e47ddfefc92e)
![capture_20240619_164649](https://github.com/valinet/ExplorerPatcher/assets/2345512/fa3d4b2e-1abd-4fc1-9b1e-940a7a9d9b5a)
![capture_20240619_164650](https://github.com/valinet/ExplorerPatcher/assets/2345512/cac86f65-b8e6-4b56-9039-fdf972199ff8)
![capture_20240619_164651](https://github.com/valinet/ExplorerPatcher/assets/2345512/dadeae11-6a54-4cca-8ac2-02625376d948)
![capture_20240619_164652](https://github.com/valinet/ExplorerPatcher/assets/2345512/75d3cdc4-7cc8-41c8-9706-2b409cce8119)
![capture_20240619_164653](https://github.com/valinet/ExplorerPatcher/assets/2345512/986ef0d5-481d-42a3-a8fd-21759e55b4ac)
![capture_20240619_164654](https://github.com/valinet/ExplorerPatcher/assets/2345512/b615ceb8-d051-4902-9766-1be209cee1e5)

**Comment by pyrates999** [comment]
Try installing the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b

**Comment by larsen0815** [comment]
Installed 22621.3527.65.5, restarted File Explorer (to be sure), but the Win+X menu still shows both entries.

**Comment by pyrates999** [comment]
Ok, please wait for a developer to respond.

**Comment by grasdk** [comment]
Same issue, and Command Prompt shortcuts are missing, even though their shortcuts are in the folder `%LocalAppdata%\Microsoft\Windows\WinX`
